packages/trip-details/src/index.tsx

```diff
@@ -330,6 +338,32 @@ export function TripDetails({
             }
           />
         )}
+        {co2 > 0 && co2Config.enabled && (
+          <TripDetail
```

Comment:
Do we want to add an info button that links to the resource we used to generate the values? Or maybe just explains the formula or something?

Reply:
Added!

Reply:
And added a link to the spreadsheet containing the formula.